simphony/io/file_mesh.py

```diff
@@ -0,0 +1,736 @@
+
+    Provides the column definition to store Points
+    forming a face and its data.
+
+    """
+
+    uuid = tables.StringCol(32, pos=1)
+    points_uuids = tables.StringCol(
+        32, pos=2, shape=(MAX_POINTS_IN_FACE,)
+        )
+
+
+class _CellDescriptor(tables.IsDescription):
+    """ Descriptor for storing Cell information
+
+    Provides the column definition to store Points
+    forming a cell and its data.
+
+    """
+
+    uuid = tables.StringCol(32, pos=1)
+    points_uuids = tables.StringCol(
+        32, pos=2, shape=(MAX_POINTS_IN_CELL,)
+        )
+
+
+class _MeshDescriptor(tables.IsDescription):
+    """ Descriptor for storing Mesh information
+
+    Provides the column definition to store mesh data.
+
+    """
+
+    uuid = tables.StringCol(32, pos=1)
+
+
+class FileMesh(object):
+    """ FileMesh.
+
+    Interface of the mesh file driver.
+    Stores general mesh information Points and Elements
+    such as Edges, Faces and Cells and provide the
+    methods to interact with them. The methods are
+    divided in four diferent blocks:
+
+    (1) methods to get the related item with the provided uuid;
+    (2) methods to add a new item or replace;
+    (3) generator methods that return iterators
+        over all or some of the mesh items and;
+    (4) inspection methods to identify if there are any edges,
+        faces or cells described in the mesh.
+
+    Attributes
+    ----------
+    data : Data
+        The X coordinate.
+    points : dictionary of Point
+        Points of the mesh.
+    edges : dictionary of Edge
+        Edges of the mesh.
+    faces : dictionary of Face
+        Faces of the mesh.
+    cells : dictionary of Cell
+        Cells of the mesh.
+
+    See Also
+    --------
+    get_point, get_edge, get_face, get_cell
+    add_point, add_edge, add_face, add_cell
+    update_point, update_edge, update_face, update_cell
+    iter_points, iter_edges, iter_faces, iter_cells
+    has_edges, has_faces, has_cells
+
+    """
+
+    def __init__(self, group, meshFile):
+        self._file = meshFile
+        self._group = group
+
+        if "points" not in self._group:
+            self._create_points_table()
+
+        if "edges" not in self._group:
+            self._create_edges_table()
+
+        if "faces" not in self._group:
+            self._create_faces_table()
+
+        if "cells" not in self._group:
+            self._create_cells_table()
+
+    def get_point(self, p_uuid):
+        """ Returns a point with a given uuid.
+
+        Returns the point stored in the mesh
+        identified by uuid. If such point do not
+        exists an exception is raised.
+
+        Parameters
+        ----------
+        uuid : int
+            uuid of the desired point.
+
+        Returns
+        -------
+        Point
+            Mesh point identified by uuid
+
+        Raises
+        ------
+        Exception
+            If the point identified by uuid was not found
+
+        """
+
+        try:
```

Comment:
The try: expect here will not work since there is no exception raised by the `where` method. I will suggest to use the 
for..else construct in this case see https://github.com/simphony/simphony-common/pull/31/files#diff-48635a0984ef717c0fb53f6c3d74e0fcR112 

also see http://python-notes.curiousefficiency.org/en/latest/python_concepts/break_else.html for a discussion on how it works